auto_Grandeur_ICA.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+"""
+Author: John Arnn
+Released: 2022-09-20
+Description:
+This script will collect needed ICA ids to start a pipeline analysis; build the icav2 command to start, and then start the analysis.
+If analysis fails to start use the icav2 arg that is created, then use icav2 manually to troubleshoot issue.
+Most likely: IDS incorrect; IDS not linked to project; syntax issue with command.
+EXAMPLE:
+python auto_mycosnp_ICA.py UT-VH0770-220915
+"""
+
+import os
+import sys
+import subprocess
+import json
+import re
+import pandas as pd
+import numpy as np
+import datetime
+
+# This function uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+now = datetime.datetime.now()
+print ("Current date and time : ")
+print (now.strftime("%Y-%m-%d %H:%M:%S"))
+
+bashCommand = "icav2 projects enter Testing"
+process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+process.communicate()
+
+run_name=str(sys.argv[1])
+if len(sys.argv) > 2:
+    additional_ref=str(sys.argv[2])
+else:
+    additional_ref=''
+
+bashCommand = "icav2 projectdata upload /Volumes/IDGenomics_NAS/WGS_Serotyping/%s/Sequencing_reads/Raw /%s/" % (run_name,run_name)
+process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+process.communicate()
+
+bashCommand = "icav2 projectdata list --parent-folder /%s/ --match-mode FUZZY" % run_name
+tmp= icav_out(bashCommand)
+ica_id=tmp[0].split()[4]
+
+# Create icav2 argument
+arg = "icav2 projectpipelines start nextflow UPHL-BioNGS_Granduer_V2 --input reads:%s " % ica_id
```

Comment:
Right now in my head, I'm debating whether or not `UPHL-BioNGS_Granduer_V2` should be a variable. 

If we change the workflow, we'll need to update this script anyway, but perhaps that would make it easier to update?